### PR TITLE
add code climate sample

### PR DESCRIPTION
**Pull request by lepistone**

### CODE REVIEW

File: sample_files/.codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+languages:
+  JavaScript: true
+  Python: true
+exclude_paths:
+- "__unported__/*"
+- "*/__openerp__.py"
+- "*/migrations/*"
```
Comment (pedrobaeza) [inline sample_files/.codeclimate.yml]: migrations can be interesting to test

**Comment by pedrobaeza** [comment]
Do we remove `__init__.py` also?

**Comment by bwrsandman** [comment]
@pedrobaeza `__init__.py` don't gave bad grades generally. It would be important to see if there are issues for odd `__init__.py` files

`__openerp__.py` files have been ignored due to the fact that they're mostly duplicate code and code climate suggests you refactor or make a class.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by bwrsandman** [comment]
@lepistone Could you add comments to the sample file to explain why something is ignored. This doesn't need to go in the maintainer tools script, but here it would be nice to explain the idea behind it.

*  "__unported__/*": obvious, no need for comment
* "*/__openerp__.py": No choice for duplicated code, this file is fed in an `eval`
* "*/migrations/*": Without OpenUpgrade in pip, the functions need to be copied for each migration file

**Comment by lepistone** [comment]
@bwrsandman done

**Comment by bwrsandman** [comment]
Could you add a sample badge to sample_files/README.md

**Comment by lepistone** [comment]
@bwrsandman done

**Comment by bwrsandman** [comment]
:+1:

**Pull request merged**